tests/unit_tests/cli/test_integration_cli.py

```diff
@@ -645,7 +648,10 @@ def test_that_the_model_raises_exception_if_active_less_than_minimum_realization
 
 
 @pytest.mark.usefixtures("copy_poly_case")
-def test_that_the_model_warns_when_active_realizations_less_min_realizations():
+@pytest.mark.scheduler
+def test_that_the_model_warns_when_active_realizations_less_min_realizations(
+    monkeypatch, try_queue_and_scheduler
```

Comment:
Maybe we should put the new fixtures in the mark above

Reply:
Hmm, should be its own PR if so.